spec/factories/topics.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  image      :string
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+FactoryBot.define do
+  test_image = Base64.encode64(File.binread("app/assets/test_image.png"))
```

Comment:
Probably we don't need to generate topics with an image, and if that's the case, I'll move the test image to another place inside `spec` folder, something like `spec/fixtures/test_image.png`